packages/discord.js/src/client/actions/MessagePollVoteAdd.js

```diff
@@ -26,6 +43,8 @@ class MessagePollVoteAddAction extends Action {
      */
     this.client.emit(Events.MessagePollVoteAdd, answer, data.user_id);
 
+    const { poll } = message;
```

Comment:
this feels a bit silly
```suggestion
    return { poll: message.poll };
```